### PR TITLE
Static root

### PR DESCRIPTION
**Pull request by ramarrom**

### CODE REVIEW

File: src/bornhack/environment_settings.py.dist.dev [hunk 1]
```diff
@@ -32,6 +32,9 @@ CAMP_REDIRECT_PERCENT = 40
 MEDIA_ROOT = os.path.join(
     os.path.dirname(os.path.dirname(os.path.dirname(__file__))), "media"
 )
+STATIC_ROOT = os.path.join(
```
Comment (valberg) [inline src/bornhack/environment_settings.py.dist.dev:35]: I think we should stick to using `pathlib.Path` as in settings.py

**Comment by valberg** [comment]
Hey @ramarrom! Great work with all the PRs - awesome to get some new energy in the project!

What is the reasoning behing this PR?

**Comment by ramarrom** [comment]
> Hey @ramarrom! Great work with all the PRs - awesome to get some new energy in the project!
> 
> What is the reasoning behing this PR?

1. Match the MEDIA_ROOT setting.
2. Make it more convenient for me to set a different STATIC_ROOT.

File: src/bornhack/environment_settings.py.dist.dev (hunk 1 above)
Comment (ramarrom) [reply src/bornhack/environment_settings.py.dist.dev:35]: That's fine

**Comment by tykling** [comment]
Just use a symbolic link if you want `static` elsewhere :+1:

**Pull request closed without merging**